service/qos/qos_common.cc

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2020 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "qos_common.hh"
+namespace qos {
+
+bool operator==(const service_level_options &lhs, const service_level_options &rhs) {
+    return &lhs < &rhs;
```

Comment:
It looks wrong. I guess you meant `==` instead of `<` ?

Reply:
yup, definitely

Reply:
Also, this comparison operator is actually unnecessary in open-source (it's already removed in a series that sits on top of this one: 8f4d1a8d40bdbfd820b53429cd53a2a5dbcb71ca ), so I'll actually rework this port to get rid of this operator entirely.